diff --git a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
index 965b24b..f3932c2 100644
--- a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
+++ b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/compile.js
@@ -35,9 +35,9 @@ async function getConfigFile(options) {
     }
     let path = await (0, find_up_1.default)(options.config);
     if (path && path.length) {
-        if (process.platform === "win32") {
+        /*if (process.platform === "win32") {
             path = `file:///${path}`;
-        }
+        }*/
         const configModule = await Promise.resolve().then(() => __importStar(require(path)));
         return configModule.default || configModule;
     }
diff --git a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/webpack.js b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/webpack.js
index 001b495..ee1f13a 100644
--- a/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/webpack.js
+++ b/node_modules/@webextension-toolbox/webextension-toolbox/dist/common/webpack.js
@@ -171,6 +171,7 @@ async function webpackConfig({ src = "app", target = "build/[vendor]", packageTa
                 ignore: copyIgnore,
             },
             to: resolvedTarget,
+            noErrorOnMissing: true,
         },
     ];
     // Copy language files (_locales) if they exist
diff --git a/node_modules/@webextension-toolbox/webextension-toolbox/src/common/compile.ts b/node_modules/@webextension-toolbox/webextension-toolbox/src/common/compile.ts
index 23f9998..5f81848 100644
--- a/node_modules/@webextension-toolbox/webextension-toolbox/src/common/compile.ts
+++ b/node_modules/@webextension-toolbox/webextension-toolbox/src/common/compile.ts
@@ -30,9 +30,9 @@ async function getConfigFile(options: CompileOptions): Promise<UserWebpack> {
   let path = await findUp(options.config);
 
   if (path && path.length) {
-    if (process.platform === "win32") {
+    /*if (process.platform === "win32") {
       path = `file:///${path}`;
-    }
+    }*/
     const configModule = await import(path);
     return configModule.default || configModule;
   }
diff --git a/node_modules/@webextension-toolbox/webextension-toolbox/src/common/webpack.ts b/node_modules/@webextension-toolbox/webextension-toolbox/src/common/webpack.ts
index 6bc68eb..80e71d0 100644
--- a/node_modules/@webextension-toolbox/webextension-toolbox/src/common/webpack.ts
+++ b/node_modules/@webextension-toolbox/webextension-toolbox/src/common/webpack.ts
@@ -222,6 +222,7 @@ export default async function webpackConfig({
         ignore: copyIgnore,
       },
       to: resolvedTarget,
+      noErrorOnMissing: true
     },
   ];
 
